components/listview/manual-operations.md

```diff
@@ -21,35 +21,29 @@ In this article you will find examples how to:
 This is, effectively, loading data on demand only when the user goes to a certain page, as opposed to the default case where you fetch all the data items initially.
 
 To implement your own paging in the listview, you need to:
-* Handle the `OnRead` event and set the current page of data to the listview's `Data` parameter
-* Set the `TotalCount` parameter of the listview to the total number of items from the data source, so it can show the proper pager.
+* Handle the `OnRead` event and...
```

Comment:
and?

Reply:
"and" refers to the following two items, but I will change this, since it may be confusing.